src/gui/mzroll/samplertwidget.cpp

```diff
@@ -32,16 +32,17 @@ AlignmentPolyVizDockWidget::AlignmentPolyVizDockWidget(MainWindow *mw) :
     toolBar->addWidget(btnHide);
 
     setTitleBarWidget(toolBar);
+
+    setXAxis();
+    setYAxis();
 }
 
-AlignmentPolyVizDockWidget::~AlignmentPolyVizDockWidget()
+SampleRtWidget::~SampleRtWidget()
 {
     delete ui;
 }
 
-void AlignmentPolyVizDockWidget::plotGraph() {
-
-    if (!_mw->alignmentPolyVizDockWidget->isVisible()) return;
```

Comment:
This check was added to prevent ui freeze in case of checking/unchecking samples when there are 100 samples loaded in El-MAVEN. The plots are being updated even when the widget is closed. Will this case be handled in this PR?

Reply:
Thanks for pointing this out. 
I'll put it back

Reply:
Done